assets/scripts/utils/array.js

```diff
@@ -1,101 +1,124 @@
-import { isArray } from './is';
+import { isArray } from './is'
 
-export function addToArray ( array, value ) {
-    const index = array.indexOf( value );
 
-    if ( index === -1 ) {
-        array.push( value );
+/**
+ * Add item of array if doesn't exist
+ * @param {array}       array   - array of items
+ * @param {?string}     value   - value to add to the array
+ * @return {void}
+ */
+
+const addToArray = (array, value) => {
+    if (array.indexOf(value) === -1) {
+        array.push(value)
     }
 }
 
-export function arrayContains ( array, value ) {
-    for ( let i = 0, c = array.length; i < c; i++ ) {
-        if ( array[i] == value ) {
-            return true;
-        }
-    }
 
-    return false;
-}
+/**
+ * Check if array contains a value
+ * @param {array}       array   - array of items
+ * @param {?string}     value   - value to find in the array
+ * @return {boolean}    true if array contains value
+ */
 
-export function arrayContentsMatch ( a, b ) {
-    let i;
+const arrayContains = (array, value) => array.find(item => item === value)
```

Comment:
This should be replaced with `Array.prototype.includes()`.

Reply:
Probably to delete too